packages/vue-app/types/vue.d.ts

```diff
@@ -25,6 +25,8 @@ declare module "vue/types/options" {
 declare module "vue/types/vue" {
   interface Vue {
     $nuxt: {
+      $isOnline: boolean;
```

Comment:
`$` prefix is not needed
It's `$nuxt.$loading`, `$nuxt.isOffline` and `$nuxt.isOnline` unless you rename them with the prefix in your changes 😜

And I suggest puting offline before online to keep alphabetical order ;)

Reply:
Agree for alaphabetical order.
About $loading, it’s because it’s a ref of the Nuxt-loading instance exposing methods :)

Reply:
You can update the typedefs with that : https://github.com/nuxt/nuxt.js/compare/dev...kevinmarrec:nuxt-typedef-example

=> This way people will have the right autocompletion on `$nuxt` including all Vue instance properties/methods

Reply:
You are awesome.